### PR TITLE
Issue #6469 fixed

### PR DESCRIPTION
**Pull request by Sanjaybhattwebkul**
## Issue Reference
#6469
>Translation added catalog configuration

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/lang/en/app.php [hunk 1]
```diff
@@ -1614,8 +1614,8 @@
             'accepted-currencies-info'              => 'Add currency code comma seperated e.g. USD,INR,...',
             'buy-now-button-display'                => 'Allow customers to directly buy products',
             'show-search-input-field'               => 'Show Search Input Field',
-            'allow-no-of-new-product-homepage'      => 'Allowed No of New Product in Homepage',
-            'allow-no-of-featured-product-homepage' => 'Allowed No of Featured Product in Homepage',
+            'allow-no-of-new-product-homepage'      => 'Allowed No. of New Product in Homepage',
```
Comment (ghermans) [inline packages/Webkul/Admin/src/Resources/lang/en/app.php:1617]: Please change this to 'Total number of new products on homepage'

File: packages/Webkul/Admin/src/Resources/lang/en/app.php [hunk 2]
```diff
@@ -1614,8 +1614,8 @@
...
+            'allow-no-of-featured-product-homepage' => 'Allowed No. of Featured Product in Homepage',
```
Comment (ghermans) [inline packages/Webkul/Admin/src/Resources/lang/en/app.php:1618]: Please change this to 'Total number of featured products on the homepage'

**Review by ghermans — CHANGES_REQUESTED** [review]
Please check the following

**Comment by Sanjaybhattwebkul** [comment]
@ghermans 
Changes **complete**, please **check**

**Pull request merged**